xhyve.xcodeproj/project.pbxproj

```diff
@@ -717,6 +718,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 3F3FF9E31BF7C5D5004C89A1 /* xhyve.xcconfig */;
 			buildSettings = {
+				ARCHS = x86_64;
```

Comment:
Xcode 12.2 Beta 4 includes arm64 as a default target arch, which xhyve does not seem to support and causes build errors.

Reply:
Do you think you could put this change in the xhyve.xcconfig?

Reply:
will do!